components/fetch-build-scan-data-cmdline-tool/src/main/java/com/gradle/enterprise/cli/FetchBuildValidationDataCommand.java

```diff
@@ -70,9 +71,12 @@ public Integer call() {
         logFinishedFetchingAllBuildScanData();
         logFetchResults(buildValidationData, customValueKeys);
 
-        printHeader();
+        printBuildValidationHeader();
         printBuildValidationData(buildValidationData);
 
+        printBuildTimeMetricsHeader();
+        printBuildTimeMetricsData(buildValidationData);
```

Comment:
My initial thought is that fetchBuildScanData should be a `Pair` rather than a `List`.

Reply:
Ah, yes. CI/local will be different. But, IMO, printBuildTimeMetricsData should still take a Pair and not a List.

Reply:
I agree. There are quite a few places we are using a `List` where a Pair or Tuple object would suffice.

Should that be done as part of this PR?

Reply:
Let's keep it as it is in this PR.